src/components/surfaces/Accordion/Accordion.tsx

```diff
@@ -18,13 +18,14 @@ const AccordionItem: React.FC<AccordionProps> = ({
   return (
     <MuiAccordion defaultExpanded={isBoolean(defaultExpanded) ? defaultExpanded : false} {...rest}>
       <AccordionSummary variant={'standard' as AccordionVariant} expandIcon={<ExpandMoreIcon />} {...summaryProps}>
-        <Typography variant="subtitle1" color={'textPrimary'}>
-          {title}
-        </Typography>
+        {typeof title === 'string' && (
```

Comment:
I suggest you to use `isValidElement(value)` function from React, you are not covering all the cases here, what if someone sends a numeric value?
You can find an example in CardHeader title value.
https://github.com/osstotalsoft/rocket-ui-ts/blob/998911ac664ff84815a565eb3cfea27952fc0650/src/components/surfaces/Card/CardHeader/CardHeader.tsx#L23-L29

Reply:
Done